### PR TITLE
Workaround reboot_gnome

### PR DESCRIPTION
**Pull request by dzedro**
bsc#981299

[tested with sleep 60](http://10.100.51.177/tests/3569)

### CODE REVIEW

**Comment by coolo** [comment]
you can take ownership :)

**Comment by dzedro** [comment]
@coolo thx :smiley:

File: tests/x11/reboot_gnome.pm [hunk 1]
```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: rename reboot_*_pre to reboot_* - the reboot.pm just contained one line
-# G-Maintainer: Stephan Kulow <coolo@suse.de>
+# Summary: rename reboot_*_pre to reboot_* - the reboot.pm just contained one line
```
Comment (coolo) [inline tests/x11/reboot_gnome.pm]: That's not really summarizing what the test does.

Comment (dzedro) [inline tests/x11/reboot_gnome.pm]: You will like this

**Pull request merged**